### DISCUSSION TITLE
Question re: coloring with user CSS

### DISCUSSION THREAD

**Discussion by Grydpype** [ransome1/sleek#711] [Q&A]
I would like to assign separate colours to @contexts, +projects, due dates & recurrences. I understand I can do this through an individual user-css, but I don't know any of the variables; nor am I a proficient hrrp developer.
Has anyone done this an can perhaps supply a couple of examples for me?

Thanks in advance,
Oliver.

**Comment by ransome1** [comment]
@Grydpype I'm afraid you need to be able to at least have a basic understanding for CSS in order to manipulate it.

A simple example on how to change the context coloring would be:
```
[data-todotxt-attribute] button,
[data-todotxt-attribute] button:hover {
	background: red!important;
	color: blue!important;
}

[data-todotxt-attribute].selected button {
	background: darkred!important;
	color: darkblue!important;
}
```

This would need to be placed in the [customStyles.css in your user data folder](https://github.com/ransome1/sleek/wiki/Custom-CSS).

**Reply by Grydpype** [reply]
Thanks for the feedback. I'm not a total novice; with help I've managed to taylor Joplin to suit my needs, so your reply will get me started and I can play around with it.
Thanks again for the great app!

**Reply by ransome1** [reply]
The easiest way to find the css selectors you're looking for is to use the developer tools. You can toggle them via keyboard shortcut or  menu bar
![Screenshot 2024-06-08 at 10 04 23 PM](https://github.com/ransome1/sleek/assets/11188741/78b50453-ecd8-4753-80ed-bf508401b884)

![Screenshot 2024-06-08 at 10 05 27 PM](https://github.com/ransome1/sleek/assets/11188741/955fd95c-de5a-407c-94f2-a09652ce7413)

**Reply by Grydpype** [reply]
Thanks for that; that is in fact my main problem: identifying the right selectors. Specifying the attributes after that is easier ...
